### PR TITLE
[TECH] Adding functionality for adminDistribution page

### PR DESCRIPTION
**Pull request by snaf-trip**

### CODE REVIEW

**Review by mtalstykh-exadel — DISMISSED** [review]

File: src/components/header/header.js [hunk 1]
```diff
@@ -93,7 +93,7 @@ const Header = () => {
             </>}
             { matches &&
               <Box className='logo-text-wrapper' component={Link} to={''}>
-                <img src={logo} alt='logo'/>
+                <img src={logo} alt='logo' width='60' />
```
Comment (mtalstykh-exadel) [inline src/components/header/header.js]: why do you have styles here?

File: src/pages/AdminDistribution/AdminDistribution.jsx [hunk 2]
```diff
@@ -1,19 +1,41 @@
...
+  TablePagination,
+  TableRow,
+  Select,
+  Button,
+} from '@material-ui/core';
+import { rows } from '../../testData/rowsForAdminDistribution';
+import { coaches } from './Coaches';
 import PropTypes from 'prop-types';
 import './AdminDistribution.scss';
+import { AssignTest } from './adminDistributtion';
```
Comment (mtalstykh-exadel) [inline src/pages/AdminDistribution/AdminDistribution.jsx]: function should start with lowercase

Comment (mtalstykh-exadel) [reply src/pages/AdminDistribution/AdminDistribution.jsx]: adminDistributtion inside an AdminDistribution.jsx? why the same names?

File: src/pages/AdminDistribution/AdminDistribution.jsx [hunk 3]
```diff
@@ -38,95 +60,117 @@ const AdminDistribution = (props) => {
   return (
     <div className='AdminDistribution'>
```
Comment (mtalstykh-exadel) [inline src/pages/AdminDistribution/AdminDistribution.jsx]: we improved a Layout to pass there a pageWrapperClass

File: src/pages/AdminDistribution/AdminDistribution.jsx [hunk 4]
```diff
@@ -38,95 +60,117 @@ const AdminDistribution = (props) => {
...
-                                      <option key={keysForOptions} value={coachName}>
-                                        {coachName}
-                                      </option>
-                                    );
-                                  })}
-                                </Select>
-                              ) : (
-                                null
-                              )}
+        <div className='paper'>
```
Comment (mtalstykh-exadel) [inline src/pages/AdminDistribution/AdminDistribution.jsx]: do you need an extra wrapper?

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by fshovchko — APPROVED** [review]

**Pull request merged**